releasenotes/notes/fix-6f81ff164bc42b89.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix issue with process agent where DD_PROCESS_AGENT_ENABLED behaivour where 'false' did not turn off process/container collection.
```

Comment:
The grammar is a little weird here, maybe `Fix issue with process agent for DD_PROCESS_AGENT_ENABLED behaivour.....`?

Reply:
Whoops - yeah I'll fix